tree/dataframe/src/RResultPtr.cxx

```diff
@@ -14,15 +14,15 @@
 #include <ROOT/RDF/RInterface.hxx>
 #include <ROOT/RDF/RLoopManager.hxx>
 
-using SnapshotPtr_t = ROOT::RDF::RResultPtr<ROOT::RDF::RInterface<ROOT::Detail::RDF::RLoopManager, void>>;
```

Comment:
Commit messages are to explain the why (and for complex changes the intricacies of the how) that might be lost in the code. It's not clear to me why this change was needed, e.g. if it's just to avoid duplication:

```
[DF] Clean up: remove duplicate type alias declaration
```